Core.Tags/WebScrap.Core.Tags/Creating/TagCreatorManager.cs

```diff
@@ -0,0 +1,31 @@
+namespace WebScrap.Core.Tags.Creating;
+
+internal class TagCreatorManager(
+    UnprocessedTagCreator unprocessedTagCreator,
+    ProcessedTagCreator processedTagCreator)
+{
+    internal void Run(ReadOnlySpan<char> tag)
```

Comment:
The naming of the whole Creating doesnt look meaningless.

manager.Run what?
Side effects deep inside.